app/gateways/db/memory/get_transfers.go

```diff
@@ -0,0 +1,20 @@
+package memory
+
+import "github.com/patriciapedrosaa/transfer-me/app/domain/entities"
+
+func (m MemoryStorage) GetTransfers() ([]entities.Transfer, error) {
+	var transfers []entities.Transfer
+
+	for _, v := range m.storageTransfer {
+		var transfer entities.Transfer
+
+		transfer.TransferID = v.id
+		transfer.AccountOriginID = v.accountOriginID
+		transfer.AccountDestinationID = v.accountDestinationID
+		transfer.Amount = v.amount
+		transfer.CreatedAt = v.createdAt
+
+		transfers = append(transfers, transfer)
+	}
+	return transfers, nil
+}
```

Comment:
nice work @patriciapedrosaa , this could be useful o/

although, I believe this won't be needed since we want to list transfers of a specific account in the application. What do you think?

Reply:
I agree!  I will update this point.